api/board/README.md

```diff
@@ -55,10 +55,18 @@ The `address` property is the operating-system-specific representation for that
 
 <a name="pinmode">
 The `mode` property can take the following values:
-- `"input"` for digital input (GPIO). The pin value can be 0 or 1.
-- `"output"` for digital output (GPIO). The pin value can be 0 or 1.
-- `"analog"` for analog input (AIO) that is converted to digital value.
+- `"digital-input"` for digital input (GPIO). The pin value can be 0 or 1.
+- `"digital-output"` for digital output (GPIO). The pin value can be 0 or 1.
+- `"analog-input"` for analog input (AIO) that is converted to digital value.
+- `"analog-output"` for analog output (AIO) that is converted from digital value.
 - `"pwm"` for PWM analog output.
```

Comment:
In my understanding PWM works with a digital output (0 or 1) alternated with a given frequency and duty cycle. An analog out pin exposes an analog voltage (rail to rail) based on D/A conversion of a digital sample series/stream.
We could remove the latter for now, but even then the name `analog-input` would stay in order to maintain forward compatibility and correctness of classification.